app/assets/stylesheets/logixRelease.scss

```diff
@@ -349,3 +349,169 @@ a {
   border: 1.5px solid $button-border-color;
   color: $black;
 }
+
+// New UI Example Page
+
+.heading {
+  color: $teal;
+  margin-bottom: 10px;
+  margin-top: 40px;
+}
+
+.row-examples {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1344px;
+  text-align: center;
+}
+
+.btn-examples{
+  background-color: $green;
+  border-radius: 30px;
+  color: $white;
+  margin: 30px 10px;
+  text-align: justify;
+  transition: all .3s ease 0s;
+}
+
+.btn-examples-light{
+  background-color: $gray-100;
+  border-radius: 30px;
+  color: $white;
+  margin: 30px 10px;
+  text-align: justify;
+  transition: all .3s ease 0s;
+}
+
+.btn-examples-view{
+  background-color: $green;
+  color: $white;
+  float: right;
+  font-size: 12px;
+  margin-bottom: 12px;
+  margin-top: 12px;
+  padding-bottom: 4px;
+  padding-left: 6px;
+  padding-right: 6px;
+  padding-top: 4px;
+  text-align: justify;
+  transition: all .3s ease 0s;
+}
+
+.btn-examples:hover{
+  background-color: $teal;
+  box-shadow: 0px 8px 15px $shadow-black;
+  color: $white;
+  transform: translateY(-7px);
+}
+
+.btn-examples-light:hover{
+  background-color: $green;
+  box-shadow: 0px 8px 15px $shadow-black;
+  color: $white;
+  transform: translateY(-7px);
+}
+
+.modal-title {
+  color: $green;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+// Styling for grid
+
+.container-examples{
+  background-color: $gray-100;
+  font-family: Open Sans;
+  padding-top: 50px;
+}
+
+.grid-cell{
+  background-color: $white;
+  border: 1px solid $teal;
+  margin-bottom: 60px;
+  margin-top: 5px;
+  max-width: 90%;
+}
+
+.grid-image{
+  background-color: $white;
+}
+
+.grid-content{
```

Comment:
Grid is too generic, since you are using these exclusively for examples, can we make this specific